<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemotesWdgt</class>
 <widget class="QWidget" name="RemotesWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>81</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Git Remote</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="txtRemotes">
     <property name="toolTip">
      <string>&lt;p&gt;Select the Git Remote, you like to edit.&lt;br/&gt;&lt;/p&gt;
&lt;p&gt;Tip: Double-Click to rename a selected Remote.&lt;/b&gt;
</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btnAddRemote">
     <property name="toolTip">
      <string>Add a new Remote.</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnDeleteRemote">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Delete this Remote.</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>URL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>txtUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="LineEdit" name="txtUrl">
     <property name="mandatory" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>libMacGitverCore/Widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
